src/Simplex/Chat/Store/Messages.hs

```diff
@@ -563,94 +584,56 @@ getDirectChatPreviews_ db user@User {userId} = do
             )
             OR c.connection_id IS NULL
           )
+          AND (
+            ct.local_display_name LIKE '%' || :search || '%'
+            OR cp.display_name LIKE '%' || :search || '%'
+            OR cp.full_name LIKE '%' || :search || '%'
+            OR cp.local_alias LIKE '%' || :search || '%'
+            OR c.local_alias LIKE '%' || :search || '%'
```

Comment:
Need to double check, but connection alias is only used until contact profile is received, after which it's transferred onto profile and afterwards alias is only updated for profile. So likely search shouldn't be made by connection alias because it could've changed on profile by then. (Connection alias may still have to be used in search when loading pending contact connections previews)